src/detext/run_detext.py

```diff
@@ -114,6 +115,8 @@ def add_arguments(parser):
     # Misc
     parser.add_argument("--random_seed", type=int, default=1234, help="Random seed (>0, set a specific seed).")
     parser.add_argument("--steps_per_stats", type=int, default=100, help="training steps to print statistics.")
+    parser.add_argument("--num_eval_round", type=int, default=None, help="number of evaluation round,this "
```

Comment:
maybe `num_eval_rounds`?

Reply:
plz update the description of how this param is used. e.g.,  "will override steps_per_eval as max(1, num_train_steps / num_eval_round)"

Reply:
> maybe `num_eval_rounds`?

Yes, this will be better

> plz update the description of how this param is used. e.g., "will override steps_per_eval as max(1, num_train_steps / num_eval_round)"

thanks, will update it